application/src/main/resources/thingsboard.yml

```diff
@@ -250,11 +250,11 @@ cassandra:
   password: "${CASSANDRA_PASSWORD:}"
   # Astra DB connect https://astra.datastax.com/
   cloud:
-    # /etc/thingsboard/astra/secure-connect-thingsboard.zip
+    # To connect to your Astra DB database using the drivers, download the secure connect bundle from the DataStax Astra Portal that contains the connection credentials. Example path is /etc/thingsboard/astra/secure-connect-thingsboard.zip
```

Comment:
Please remove the " Example path is /etc/thingsboard/astra/secure-connect-thingsboard.zip"